apps/realworld/lib/realworld/application.ex

```diff
@@ -7,12 +7,7 @@ defmodule Realworld.Application do
 
   @impl true
   def start(_type, _args) do
-    children = [
```

Comment:
why are we removing this?

Reply:
Sorry github is not helping me to see which line you are mentioning